content/tutorial-react-native/rn-03.md

```diff
@@ -0,0 +1,343 @@
+# Tutorial 03 - Advanced Queries
+
+This is the third exercise in the **React Native Track** of this Apollo Client Tutorial! 
+If you prefer React or Angular 2 over React, head over to the respective tutorial track.
+
+## Goal
+
+In this exercise we will have a look at advanced query features with the **goal** of showing a list of pokemon that our 
+trainer owns. Upon clicking on a pokemon in this list, we will see a detailed view of that pokemon and can navigate back
+to the complete pokedex.
+
+## Introduction
+
+Move to the third exercise of the Pokedex React App and install the dependencies from your console
+
+```sh
+cd pokedex-react-native/exercise-03-initial
+yarn install
+```
+
+## Display a list of pokemon with advanced queries
+
+Before we start working directly on our goal to show the pokemons a trainer owns, let's take some time to get more 
+familiar with some of the available options when using queries.
+
+### Static Query Variables
+
+One of the available query options are variables. A common use case for query variables is when we want to display 
+the same type of information for two different nodes. We will see that use case shortly, but for now we are introducing 
+a query variable to the `TrainerQuery`. This is how it looked like at the end of the last exercise:
+
+```js
+const TrainerQuery = gql`query {
+  Trainer(name: "__NAME__") {
+     name
+   }
+ }`
+```
+
+To introduce a variable for the trainer `name`, we have to add the `$name` argument to the query parameters and assign 
+it to the `name` argument of `Trainer`:
+
+```js
+const TrainerQuery = gql`query($name: String!) {
+  Trainer(name: $name) {
+    name
+  }
+}`
+```
+
+Note that we have to denote the variable type as well, `String!` signifying a required String in this case. Of course, 
+now we also have to supply a value for that variable when we use it to wrap the `Pokedex` component:
+
+```js
+export const TitleWithData = graphql(TrainerQuery, {
+options: {
+    variables: {
+      name: "Tycho"
+    }
+  }
+})(Title)
+```
+
+### Nested Queries
+
+Now that we saw query variables in action we can focus on displaying the pokemons of a given trainer. We will use the a 
+React Native ListView as it is the most pragmatic way to display a list of element in a movile app, while being 
```

Comment:
`to display a list of element in a movile app` => `to display a list of elements in a mobile app`